.package.resolved

```diff
@@ -246,11 +246,11 @@
       },
       {
         "package": "Zip",
-        "repositoryURL": "https://github.com/marmelroy/Zip.git",
+        "repositoryURL": "https://github.com/maparoni/Zip.git",
```

Comment:
let's watch [this](https://github.com/marmelroy/Zip/pull/221) PR and the repo, so we can in the future move back to the official release instead of a fork